airlock-dev-apache-ranger/resources/servicedef/ranger-servicedef-s3.json

```
@@ -103,10 +103,18 @@
     [
       {
         "itemId": 1,
-        "name": "cidr",
-        "label": "IP address within cidr range?",
-        "description": "Ip address within cidr range?",
-        "evaluator": "com.ing.wbaa.ranger.plugin.conditionevaluator.IpCidrMatcher",
+        "name": "cidrAllIPs",
```

wouldn't be the name misleading?  eg `cidrAllIPs:192.0.0.0/24,168.0.0.0/24` The **All** could be related to ips written in ranger not the user ips we want to check (???). 
```suggestion
        "name": "cidrAllUserIPs",
```
unfortunately - the truth is that the longer the name is the worse :( so if it is no misleading for you just leave it.

---

Or CidrRequireAllIps

---

CidrRequireAllIps doesn't really solve the problem IMHO. I used the name suggested by Krzysztof and updated the description as well.